kf_lib_data_ingest/etl/load/load.py

```diff
@@ -299,6 +298,35 @@ def _submit(self, entity_id, entity_type, endpoint, body):
             self.logger.debug(f'Response error:\n{pformat(resp.__dict__)}')
             raise RequestException(resp.text)
 
+    def _apply_value_transform(self, schema, payload):
```

Comment:
can we call this...uhh.............
_apply_target_value_type_schema

Reply:
I don't know what that name means. This function is basically just applying whatever function is supplied in the target api config for a particular property. That function can be a type mapping but it doesn't really have to be. It could be anything that takes the original property value and does something to it